res/xml/devanagari.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard>
+  <row>
```

Comment:
This row contains only the the consonants https://omniglot.com/language/articles/devanagari.htm